packages/transformer/src/IModelTransformer.ts

```diff
@@ -2221,10 +2221,15 @@ export class IModelTransformer extends IModelExportHandler {
    * It is public for unsupported use cases of custom synchronization transforms.
    * @note if [[IModelTransformOptions.argsForProcessChanges]] are not defined in this transformation, this will fail
    * without setting the `force` option to `true`
+   * The saveReverseVersion is added to save the reverse sync version for processAll transformations, when set to `true`, reverse sync version is saved
    */
-  public updateSynchronizationVersion({ force = false } = {}) {
+  public updateSynchronizationVersion({
+    force = false,
+    saveReverseVersion = false,
+  } = {}) {
     const notForcedAndHasNoChangesAndIsntProvenanceInit =
       !force &&
```

Comment:
is it needed to check both `force` and 'saveReverseVersion' here? Maybe checking just `force` is enough here..?

Reply:
if we expect to set force to true every time, we use saveReverseVersion, then maybe, the additional check might not be needed. I guess the point of setting force to true is so that we don't return early? So adding saveReverseVersion in that check might be unnecessary

Reply:
If someone is passing force does that mean they also want to saveReverseVersion in all cases?

Reply:
Clarified offline